### ISSUE TITLE
[accessibility][voice reader] Mac Voice Over reads all elements within a Dialog component instead of only reading the highlighted element 

### ISSUE THREAD

**Issue by lutorre** [OfficeDev/office-ui-fabric-react#10700]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: Google chrome - MacOS Mojave 10.14.5

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

After opening a Dialog (while having voice over activated) that contains selectable elements (button, radio buttons, text fields, etc), the focus goes to the first element within the dialog as expected, but voice reader reads all elements (not only the focused one)

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/lutorre/pen/QWWLWNY?&editable=true

#### Actual behavior:

Voice reader reads all elements (not only the focused one)

#### Expected behavior:

Voice reader reads only the focused element

**Comment by khmakoto** [comment]
Hi @lutorre, thanks for filing this issue! We'll take a look to see what the next steps are as soon as we can.

**Comment by khmakoto** [comment]
Hi @lutorre, I just checked the codepen you provided (thank you for that!) with Edge and Narrator and I can't seem to repro the bug you're having with VoiceOver on the Mac. Since our screen reader support is limited to Edge + Narrator and it seems that we're using the correct aria values here my recommendation would be to provide feedback to VoiceOver.

Let me know if there is anything else I can help you with!
